tests/content/viewing-activity-parser-test.js

```diff
@@ -8,36 +9,36 @@ describe('ViewingActivityParser', function() {
 
   describe('parse', function() {
     it('creates the activity when synced_at is undefined', function() {
-      var activity = ViewingActivityParser.parse(new Date(undefined), window.movieActivity);
+      var activity = ViewingActivityParser.parse(moment(undefined, "YYYY-MM-DDTHH:mm:ss.SSSSZ"), window.movieActivity);
       expect(activity instanceof ViewingActivity).toBeTruthy();
     });
 
     it('creates the activity when synced_at equals to item date', function() {
-      var activity = ViewingActivityParser.parse(new Date('10/2/2015'), window.episodeActivity);
+      var activity = ViewingActivityParser.parse(moment('10/2/2015', 'M/D/YYYY'), window.episodeActivity);
       expect(activity instanceof ViewingActivity).toBeTruthy();
```

Comment:
Ahh thanks. I belive this is another timezone issue. I've put the time in absolute time in netflix-viewing-activity, while this might both be before and after the original date depending on location. Thanks.